WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift

```diff
@@ -96,6 +96,7 @@ public class AppSettingsViewController: UITableViewController {
         return {
             value in
             MediaSettings().maxImageSizeSetting = value
+            ShareExtensionService.configureShareExtensionMaximumMediaDimension(value)
```

Comment:
I think it's okay, since it's the spot in which MediaSettings is updated. It's either this one, or within the `maxImageSizeSettings` didSet / setter. But this one _feels_ cleaner!